app/models/dependent.rb

```diff
@@ -47,6 +47,8 @@
 #
 
 class Dependent < ApplicationRecord
+  include SoftDeletable
```

Comment:
oh smart. this is great that we can use the same pattern across multiple tables.